omod/src/main/webapp/pages/account/resetPassword.gsp

```diff
@@ -0,0 +1,57 @@
+<!-- This is the login page -->
+<%
+    ui.includeFragment("appui", "standardEmrIncludes")
+    ui.includeCss("pihcore", "account.css")
+    ui.includeJavascript("pihcore", "account/changePassword.js")
+%>
+
+<!DOCTYPE html>
+<html>
+<head>
+    <title></title>
+    <link rel="shortcut icon" type="image/ico" href="/${ ui.contextPath() }/images/openmrs-favicon.ico"/>
+    <link rel="icon" type="image/png\" href="/${ ui.contextPath() }/images/openmrs-favicon.png"/>
+    ${ ui.resourceLinks() }
+</head>
+<body>
+<script type="text/javascript">
+    var OPENMRS_CONTEXT_PATH = '${ ui.contextPath() }';
+    var errorMessageOldPassword = "${ui.message("emr.account.changePassword.oldPassword.required")}";
+    var errorMessageNewPassword = "${ui.message("emr.account.changePassword.newPassword.required")}";
+    var errorMessageNewAndConfirmPassword = "${ui.message("emr.account.changePassword.newAndConfirmPassword.DoesNotMatch")}";
```

Comment:
Are these used?

Reply:
So these are used?  Seems not sure why the old password would be required on this page since it isn't known?